lib/palletjack/tool.rb

```diff
@@ -50,7 +50,17 @@ class Tool
     #   MyTool.run { jack.each(kind:'system') {|sys| puts sys.to_yaml } }
```

Comment:
Documentation and examples should be updated to describe the new API and deprecation of the old.